tests/zfs-tests/tests/functional/io/io_uring.ksh

```diff
@@ -44,6 +44,11 @@ if ! $(grep -q "CONFIG_IO_URING=y" /boot/config-$(uname -r)); then
 	log_unsupported "Requires io_uring support"
 fi
 
+# Disabled for the CentOS 9 kernel
+if [[ $(linux_version) -eq $(linux_version "5.14") ]]; then
+	log_unsupported "Disabled on CentOS 9, fails with 'Operation not permitted'"
+fi
```

Comment:
If you wanted to check for the OS version itself:
```sh
if [ -e /etc/os-release ] ; then
       source /etc/os-release
       if ((floor($REDHAT_SUPPORT_PRODUCT_VERSION) == 9)) ; then
              log_unsupported "Disabled on CentOS 9, fails with 'Operation not permitted'"
       fi
fi
```

Reply:
Good thought, that's more targeted.  Updated.